database/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM postgres:9.6
+FROM postgres:11
```

Comment:
we should uplift to version 16 once migration to azure Postgres flexi server completes